src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-260046.mdx

```diff
@@ -16,7 +16,7 @@ For a list of known issues with the Python agent, see [Status of the Python agen
 
 ### New Feature
 
-* Custom Insights Events
+* Custom Events
```

Comment:
Should we really be retroactively changing release notes?

Reply:
yeah probably not, you're right. i started changing them then stopped after a couple.